packages/stark-core/src/common/translations/translations/nl.ts

```diff
@@ -27,6 +27,34 @@ export const translationsNl: object = {
 			SEARCH: "Zoeken",
 			NEW_ITEM: "Nieuw",
 			SAVE_AND_NEXT: "Bewaar en volgende"
+		},
+		MULTI_COLUMN_SORTING: {
+			TITLE: "Sorteren van meerdere kolommen",
+			ADD_SORTING_LEVEL: "Voeg sorteringsniveau toe",
+			REMOVE_SORTING_LEVEL: "Verwijder sorteringsniveau",
+			SORT_BY: "Sorteer op",
+			THEN_BY: "Dan op",
+			CANCEL: "Annuleer",
+			SAVE: "Opslaan"
+		},
+		TABLE: {
+			NB_SELECTED_ROWS: "Geselecteerd",
+			TOGGLE_SELECTION: "Selectie omkeren",
+			SELECT_ALL: "Alles selectioneren",
+			DESELECT_ALL: "Selectie wissen",
+			SELECT_DESELECT_ALL: "Alles selectioneren/Selectie wissen",
+			MORE: "Meer",
+			INDEX: "Index",
+			ACTIONS: "Acties",
+			FILTER: "Filter (max 20 tekens)",
+			CLEAR_FILTER: "Clear filter",
```

Comment:
Can you please put the right translation for this new key?

Reply:
Well for the french part I can handle it, but what about NL? 
Isn't it better to create a ticket about it and before releasing to prod a dutch speaker can translate what we need?

Reply:
I think you can ask @RobbyDeLaet for this one, since it is just one line ;)